genbindings.sh

```diff
@@ -48,21 +48,21 @@ fi
 # Finally, sanity-check the generated C and C++ bindings with demo apps:
 
 # Naively run the C demo app:
-gcc -Wall -g -pthread demo.c ../target/debug/liblightning.a -ldl
+gcc -Wall -g -pthread demo.c target/debug/liblightning.a -ldl
```

Comment:
What do you think about splitting out the names, and paths into variables? I don't expect this file to change all that much, but it deduplicates a lot of the path names.

Reply:
Sure, I mean happy to do whatever, not sure its gonna change much independently though.